src/Timer/ProgressReporter.cc

```diff
@@ -0,0 +1,31 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
```

Comment:
I think that this should be called `ListProgressReporter`, since it is specific to list progress reports.